app/controllers/commitments_controller.rb

```diff
@@ -18,8 +18,9 @@ def index
       filter_params[:filters] = []
       filter_params[:filters] << params[:country_filters]
     end
+
     @paginatedCommitments = Commitment.paginate_commitments(filter_params.to_json).to_json
-    @commitments_count = Commitment.where(state: 'live').count
+    @commitments_count = Commitment.published.count
```

Comment:
Should apply this refactor to line 8 of the country_commitments_controller too

Reply:
changed